app/lib/common/widgets/sliver_scaffold.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:acter/common/themes/app_theme.dart';
 import 'package:flutter/material.dart';
 
-class SideSheet extends StatelessWidget {
+class SliverScaffold extends StatelessWidget {
```

Comment:
Sorry but I doesn't really understand the concept of `SliverScaffold`.

Reply:
No issue with wording for now as par as it doesn't effecting any functional changes.